tests/cases/compiler/exportAsNamespace_augment.ts

```diff
@@ -0,0 +1,22 @@
+// @Filename: /a.d.ts
+export as namespace a;
+export const x = 0;
+export const conflict = 0;
+
+// @Filename: /b.ts
+import * as a2 from "./a";
+
+declare global {
+    namespace a {
+        export const y = 0;
+        export const conflict = 0;
+    }
+}
+
+declare module "./a" {
+    export const z = 0;
+    export const conflict = 0;
+}
+
+a.x + a.y + a.z + a.conflict;
```

Comment:
how can we access `a` in a module?

Reply:
`a` is a global namespace (because of `declare global {}`), so IMO no issue.